src/components/Cart/cart.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import {Card, CardTitle, CardText,Button} from 'reactstrap';
+import {DISHES} from '../../shared/dishes';
+class Cart extends Component {
+    constructor(props){
+        super(props);
+        console.log(this.props.dishSelected);
+        this.state={
+            dishes: DISHES,
+        }
+    }
+
+    componentDidMount()
+    {
+        this.setState({dishes: DISHES});
+        console.log("Component Mounted"+this.props.dishSelected);
+    }
+
+    checkIndex(dish)
+    {
+        var index=this.props.dishSelected.indexOf(dish.id);
+        if(index<0){
+            return false;
+        }
+        else{
+            return true;
+        }
+    }
+
+    renderHeading()
+    {
+        if(this.props.dishSelected.length>0)
+        {
+            return(
+                <div>
+                    <h3>Order Summary</h3>
+                </div>
```

Comment:
This should be in single or double quotes like a string.

Reply:
If I return a string my cart's main render function will be large to track. Will look into this but if you could elaborate on what I should do, it would make this more clear

Reply:
Okay will change this